apps/acrs/behaviours/declaration.js

```diff
@@ -1,6 +1,7 @@
 const Utilities = require('../../../lib/utilities');
+const config = require('../../../config');
```

Comment:
you could destructure the `dobFormat` and `AGE_LIMIT` out of the config object here like `const { dobFormat, AGE_LIMIT } = require('../../../config');

Up to you but in the lower lines it might make the parameters a bit more concise when they are used.

Reply:
cool got you